todo_app.sql

```diff
@@ -0,0 +1,74 @@
+DROP TABLE IF EXISTS tasks;
+
+CREATE TABLE tasks (
+  id serial NOT NULL PRIMARY KEY,
+  title character varying (255) NOT NULL,
+  description text,
+  created_at timestamp without time zone NOT NULL DEFAULT now(),
+  updated_at timestamp without time zone,
+  completed boolean
+);
+
+ALTER TABLE tasks 
+DROP completed;
+
+ALTER TABLE tasks 
+ADD completed_at timestamp DEFAULT NULL;
+
+ALTER TABLE tasks 
+DROP updated_at;
```

Comment:
Instead of dropping the column, you should just alter it. If this was a database that had a bunch of rows, you've just lost all the updated_at data across all the rows.